nucleus/dataset.py

```diff
@@ -1219,6 +1219,13 @@ def upload_predictions(
                     "predictions_ignored": int,
                 }
         """
+        # TEMP Workaround to avoid annotation_id conflicts across distinct model_runs
```

Comment:
Could you link the shortcut ticket in this comment as well?

Reply:
```suggestion
        # TEMP Workaround to avoid annotation_id conflicts across distinct model_runs
        # Revert once this is implemented: https://app.shortcut.com/scaleai/story/344585/relax-nucleus-model-prediction-on-conflict-condition-to-annotation-id-dataset-item-id-model-run-id
```

Reply:
Consider doing `TODO: sh344585` to shorten it.

Reply:
amended. CR within a CR 😮